andela/travel_tool_front
# #161374570 Dashboard Analytics

#### What does this PR do?
Implements trip requests analytics

#### Description of Task to be completed?
    - Create a reusable analytics card
    - Implement filtering by date range
    - Populate the cards with data from the API
    - Implement charts with real data from the API

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
```ft-dashboard-analytics-161374570```
- Install dependencies:
```yarn install```
- Start the application:
```yarn start```

- Login to the application
- Navigate to the dashboard page on the sidebar
- You should be able to get results as shown in the screenshot below.

#### Any background context you want to provide?
1. To use the Dashboard - Create your standalone component and plug it into the dashboard component.
2. You should have the user role of a Travel Administrator or Super Administrator to be able to see the `manage` page. To do that:
  - Download `Postico` and connect to your travela database with relevant details: Db name, password, port, user
  - Click `userRoles` table and set the `roleId` to that of a Travel Administrator(29187) or Super Administrator(10948) 

#### What are the relevant pivotal tracker stories?
[161374570](https://www.pivotaltracker.com/story/show/161374570)

#### Screenshots (if appropriate)
![ezgif com-crop 1](https://user-images.githubusercontent.com/22371596/47866199-23205f80-de0f-11e8-9a8b-0b23029a7aba.gif)
